fvtest/compilertest/tests/X86OpCodesTest.cpp

```diff
@@ -4748,7 +4748,6 @@ X86OpCodesTest::UnsupportedOpCodesTests()
    addUnsupportedOpCodeTest(_numberOfBinaryArgs, TR::ldiv, "lDiv", _argTypesBinaryLong, TR::Int64);
    addUnsupportedOpCodeTest(_numberOfBinaryArgs, TR::lrem, "lRem", _argTypesBinaryLong, TR::Int64);
    addUnsupportedOpCodeTest(_numberOfUnaryArgs, TR::l2a, "l2a", _argTypesUnaryLong, TR::Address);
-   addUnsupportedOpCodeTest(_numberOfTernaryArgs, TR::aternary, "aternary", _argTypesTernaryAddress, TR::Address);
```

Comment:
Please update the copyright header on this file.

Reply:
Addressed.